docs/modules/extensions/pages/processors/block-processor.adoc

```diff
@@ -0,0 +1,135 @@
+= Block Processor
+
+BlockProcessor is a SyntaxProcessor
+
+== Registration method
+
+A BlockProcessor is registered with `block`.
+
+== DSL methods
+
+include::page$processor.adoc[tag=syntaxproc]
+
+`contexts`::
+Sets the contexts allowed for this block.
+The choices are:
+
+[cols=2,opts=headers]
```

Comment:
This table should be a referenced rather than defined inline. These are well-defined contexts and they need to be defined elsewhere. They aren't specific to extensions.

Reply:
How about allowing it here for now and finding a better place for it later?  I'm not interested in rewriting all of the Asciidoc documentation for now.